### PR TITLE
Added regression tests, updated dependencies

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: test/integration/prototype-check.postman_collection.json [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
...['a', 'b', 'c'].none('d');\ntests[\"Array prototype any\"] = [ [1,2], [2,3] ].any([2,3]);\ntests[\"Array prototype average\"] = [ 1, 2, 3, 4, 5 ].average() === 3;\n\n// Extended Date prototype tests\ntests[\"Date prototype getTime\"] = Date.now()==(new Date()).getTime();\n\n// Extended Function prototype tests\nvar fCount = 0;\nvar fn = (function() {\n  fCount++;\n}).once(); fn(); fn(); fn();\ntests[\"Function prototype once\"] = fCount===1;\n\n// Extended Number prototype tests\ntests[\"Number prototype hex\"] = (56).hex() === '38';\ntests[\"Number prototype isEven\"] = (56).isEven() === true;\ntests[\"Number prototype ordinalize\"] = (56).ordinalize() === '56th';\ntests[\"Number prototype format\"] = (56789.10).format() === '56,789.1';\n\n// Extended String prototype tests\ntests[\"String prototype endsWith\"] = 'jumpy'.endsWith('py');\ntests[\"String prototype negated endsWith\"] = !('jumpy'.endsWith('MPY'));\ntests[\"String prototype camelize\"] = 'a-beta'.camelize() === 'ABeta';\ntests[\"String prototype repeat\"] = 'a'.repeat(5) === 'aaaaa';\ntests[\"String prototype shift\"] = 'abc'.shift(5) === 'fgh';\ntests[\"String prototype spacify\"] = 'a-b_cD'.spacify() === 'a b c d';"
```
Comment (czardoz) [inline test/integration/prototype-check.postman_collection.json]: Can we add more Date tests? I see just one check here :confused:

File: test/integration/regression-tests.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		{
+			"id": "13d60357-a26a-659c-9b97-bb8caca389cd",
+			"headers": "",
+			"url": "https://echo.getpostman.com/cookies/set?foo1={{envKey}}&foo2=bar",
+			"preRequestScript": null,
+			"pathVariables": {},
+			"method": "GET",
+			"data": null,
+			"dataMode": "params",
+			"tests": "var cookies = JSON.parse(responseBody).cookies;\n\ntests[\"Status code is 200\"] = responseCode.code === 200;\ntests[\"foo1 cookie is present\"] = cookies[\"foo1\"] == postman.getEnvironmentVariable(\"envKey\");",
```
Comment (czardoz) [inline test/integration/regression-tests.postman_collection.json]: Can we also add a small test for checking the cookie using `postman.getResponseCookie()`?

File: test/integration/regression-tests.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		{
+			"id": "dc930208-754a-bbac-d61e-663e7d5c1f9c",
+			"headers": "//disabled-header: randomHeaderString\n",
+			"url": "https://echo.getpostman.com/headers",
+			"preRequestScript": null,
+			"pathVariables": {},
+			"method": "GET",
+			"data": null,
+			"dataMode": "params",
+			"tests": "var headers = JSON.parse(responseBody).headers;\ntests[\"Status code is 200\"] = responseCode.code === 200;\n\n\ntests[\"Disabled header is present\"] = !headers[\"disabled-header\"];",
```
Comment (czardoz) [inline test/integration/regression-tests.postman_collection.json]: The test here should be  "Disabled header is absent" becuse it's supposed to be absent

File: test/integration/regression-tests.postman_collection.json (hunk 2 above)
Comment (kunagpal) [inline test/integration/regression-tests.postman_collection.json]: Sure.

File: test/integration/regression-tests.postman_collection.json (hunk 3 above)
Comment (kunagpal) [inline test/integration/regression-tests.postman_collection.json]: Good point.

File: test/integration/prototype-check.postman_collection.json (hunk 1 above)
Comment (kunagpal) [inline test/integration/prototype-check.postman_collection.json]: Alright.

File: README.md [hunk 4]
```diff
@@ -184,7 +184,7 @@ discontinued. For documentation on the older command options, refer to
 - `-h`, `--help`<br />
   Show commandline help, including a list of options, and sample use cases.
 
-- `-v`, `--version`<br />
+- `--version`<br />
```
Comment (shamasis) [inline README.md]: what happened to `-v`?

File: test/integration/regression-tests.postman_environment.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+			"value": "1",
+			"enabled": true
+		}
+	],
+	"timestamp": 1470909317305,
+	"synced": false,
+	"syncedFilename": "",
+	"team": null,
+	"isDeleted": false
+}
```
Comment (shamasis) [inline test/integration/regression-tests.postman_environment.json]: missing end of file newline

File: test/integration/regression-tests.postman_collection.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
+{
```
Comment (shamasis) [inline test/integration/regression-tests.postman_collection.json]: Can we please be more specific with the file name saying that these are v2 regression tests? Future regression tests will all be in their own collections.

File: README.md (hunk 4 above)
Comment (kunagpal) [inline README.md]: It is meant to be used for `verbose` logging.

Comment (shamasis) [inline README.md]: But WHY in this PR?

Comment (kunagpal) [inline README.md]: This is just a documentation update, the `-v` option was removed from the CLI argument parser elsewhere.

Comment (shamasis) [inline README.md]: Ok. But keeping them in separate PRs helps to keep track of things when we need to look historically. Anyway, please move this out of this PR and into a separate one.

File: test/integration/prototype-check.postman_collection.json (hunk 1 above)
Comment (czardoz) [inline test/integration/prototype-check.postman_collection.json]: Where are they added?

**Pull request merged**